src/include/nodes/cypher_nodes.h

```diff
@@ -195,6 +195,7 @@ typedef struct cypher_map
     ExtensibleNode extensible;
     List *keyvals;
     int location;
+    int keep_null; // if false, keyvals with null value are removed
```

Comment:
Why is this an integer and not a Boolean? It is preferable to use a Boolean type and true/false instead of 1/0.

Reply:
@jrgemignani My bad. Fixed it now.

Reply:
Using an integer in place of a Boolean can be okay in some instances. I was just wondering because it looked like it was only ever used as a Boolean.